src/components/m-datepicker/index.js

```diff
@@ -24,13 +24,13 @@ class AXADatepicker extends BaseComponentGlobal {
     classes: PropTypes.string,
     buttonOk: PropTypes.string,
     buttonCancel: PropTypes.string,
-    locale: localePropType,
+    locale: PropTypes.string,
```

Comment:
Same, doesn't check for language nor country code

Reply:
I removed all datepicker changes. they were not meant to be in this PR.

OT:
The regex in the prop check was broken.
This should be done with tests, adding a special regex prop here is a bit overkill, isn't it?

Reply:
If a regex is broken, please create an issue.
Well, overkill in which regard? If we want to pick up as many users as possible, we should provide them with all data possible.

Think of React, the do not only render DOM for you, they also warn about semantic HTML, ARIA, etc., which could be considered overkill too

Reply:
Fine for me. For me a locale is a just a special type of a concatinated string, if you put the wrong string, yes then it does not work and that's fine. ... it's not type validation it's string or input data validation here.

Reply:
@jackblackCH 
> The regex in the prop check was broken.

I'm curious in which regard it is not working, may I ask you to provide a failing example here:
https://regex101.com/r/cYG3RI/1/